packages/java/src/browser/java-client-contribution.ts

```diff
@@ -14,6 +14,10 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+// Copyright (c) Red Hat.
+// Licensed under EPL-1.0 license
+// copied and modified from https://github.com/redhat-developer/vscode-java/blob/v0.44.0/src/javaServerStarter.ts#L105-L121
```

Comment:
Can we be a bit more explicit and mention which part was copied. e.g.: 
"Function parseVMargs() copied and modified from https://github.com/redhat-developer/vscode-java/blob/v0.44.0/src/javaServerStarter.ts#L105-L121"

Reply:
@marcdumais-work 
Thanks. I fixed to use your proposal.